example-cap-server/http/http-client.env.json

```diff
@@ -1,5 +1,8 @@
 {
   "local": {
     "base_url": "http://localhost:4004"
+  },
+  "remote": {
+    "base_url": "https://example-cap-server.cfapps.eu10.hana.ondemand.com"
```

Comment:
this not correct with the random route right? Does it even make sense to check this into VCS